src/sim.rs

```diff
@@ -98,7 +116,7 @@ impl<'a> Sim<'a> {
             world.register(addr, &self.config);
```

Comment:
Can you update this event? https://github.com/tokio-rs/turmoil/blob/main/src/world.rs#L108 It could use the `nodename` instead of rendering the optional.

Reply:
like this?